generator/generator_tool/generator_tool.cpp

```diff
@@ -245,6 +249,12 @@ int main(int argc, char ** argv)
   if (FLAGS_dump_feature_names != "")
     feature::DumpFeatureNames(datFile, FLAGS_dump_feature_names);
 
+  if (!FLAGS_srtm_path.empty())
```

Comment:
This section of code works with an already complete mwm file (the code that
generates routing sections is I believe an artifact from the time when
routing information was stored separately).